doc/Learn/Tutorials/Authentication/HowTo-Authentication.md

```diff
@@ -6,7 +6,7 @@ uid: Learn.Tutorials.Authentication.HowToAuthentication
 `Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to use one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```

Comment:
Please revert changes to this file in this PR - if these changes are required, please push to a different PR